tests/devtools.test.tsx

```diff
@@ -436,43 +438,49 @@ describe('when it receives an message of type...', () => {
   })
 })
 
-it('works with redux middleware', () => {
-  const api = create(
-    devtools(
-      redux(
-        ({ count }, { type }: { type: 'INCREMENT' | 'DECREMENT' }) => ({
-          count: count + (type === 'INCREMENT' ? 1 : -1),
-        }),
-        { count: 0 }
+describe('with redux middleware', () => {
+  let api: any
```

Comment:
Do you have a better idea than `any`?

Reply:
Wouldn't `let api = create(...)` just work? Let me try

Reply:
It is because of `beforeEach`.